app/models/value_sorter.rb

```diff
@@ -3,8 +3,12 @@
 module ValueSorter
   class << self
     def sort(values, locale: "en")
-      if values.first.position.present?
-        values.sort_by(&:position)
+      return values if values.length <= 1
```

Comment:
I wonder if this optimization is worth it? Feels like it adds more complexity for fairly minimal gains? (I expect in the singular case both sorting paths should be fairly minimal overhead)

If we do keep it, one small nit since we generally prefer `size` over `length` in ruby land:
```suggestion
      return values if values.size <= 1
```

Reply:
The changes to this file are no longer needed, so I've reverted them